### ISSUE TITLE
Add retries to file copy operations to Google Storage

### ISSUE THREAD

**Issue by nhammond** [StanfordBioinformatics/loom#353]
Occasionally we see an error like this:

```
500 Backend Error (GET https://www.googleapis.com/storage/v1/b/gbsc-gcp-project-scgs-dev-loom/o/loom-data%2Fruns%2F2017-04-28T02.37.49Z-a39d417b-WES_CGS_BroadBased_SingleSample_b37_with_ACE3Benchmarking%2F0067b93e-printreads_applyBQSR_GATK%2Ftask-f34e566a%2Fattempt-5169b522%2Fwork%2F0a37990a_NA12878.recalibrated.bai)
```
It appears to be a sporadic problem on Google's side. We need to protect against it be allowing retries for unxplained 500 errors.

**Comment by nhammond** [comment]
To add context, this is the timepoint created by the taskrunner:

```
{
            "message": "Failed to copy inputs to workspace",
            "detail": "500 Backend Error (GET https://www.googleapis.com/storage/v1/b/gbsc-gcp-project-scgs-dev-loom/o/loom-data%2Fruns%2F2017-04-28T02.37.49Z-a39d417b-WES_CGS_BroadBased_SingleSample_b37_with_ACE3Benchmarking%2F0067b93e-printreads_applyBQSR_GATK%2Ftask-f34e566a%2Fattempt-5169b522%2Fwork%2F0a37990a_NA12878.recalibrated.bai)",
            "timestamp": "2017-04-28T03:22:43.934408Z",
            "is_error": true
        },
```

**Comment by nhammond** [comment]
Waiting on response to Google support ticket.

**Comment by nhammond** [comment]
This is a general problem. All calls to Google service APIs should retry.
https://github.com/StanfordBioinformatics/loom/issues/369

**Comment by nhammond** [comment]
Implemented in 7c0bb4515ae29379b8e5594dc2edd2be599878c5
